lms/static/lms/js/require-config.js

```diff
@@ -19,6 +19,12 @@
         };
         defineDependency("jQuery", "jquery");
         defineDependency("_", "underscore");
+        if (window._ && window._.str) {
```

Comment:
You could reduce code duplication by having `defineDependency` take in a predicate function to replace the call to `window[globalVariable]`, since we need to test for the presence of both `_` and `_.str`.